planner/core/logical_plan_builder.go

```diff
@@ -1972,7 +1974,7 @@ func (b *PlanBuilder) pushTableHints(hints []*ast.TableOptimizerHint, nodeType n
 			preferAggType |= preferHashAgg
 		case HintStreamAgg:
 			preferAggType |= preferStreamAgg
-		case HintIndex:
+		case HintUseIndex:
```

Comment:
What will happen if using both `ignore_index(t, idx_a)` and `use_index(t, idx_a)` in a sql block?

Reply:
Do we need to add a test for it?

Reply:
Seems that we will ignore all indexes... I'll add test and fix it.

Reply:
now it works just like normal index hint. Added some tests, you can check it out.